test/view/analysis/analysis_screen_test.dart

```diff
@@ -300,7 +300,7 @@ void main() {
 
         await tester.tap(find.byKey(const Key('goto-previous')));
         // need to wait for current move change debounce delay
-        await tester.pump(const Duration(milliseconds: 200));
+        await tester.pumpAndSettle(const Duration(milliseconds: 200));
```

Comment:
not 100% sure why `pump()` worked fine before and then with this commit `pumpAndSettle()` is needed - but with only `pump()` the `Timer` behind the debouncer never calls it callback